client/components/mma/holiday/ExistingHolidayStopActions.tsx

```diff
@@ -22,7 +27,66 @@ interface ExistingHolidayStopActionsProps extends MinimalHolidayStopRequest {
 	) => void;
 }
 
-class WithdrawHolidayStopAsyncLoader extends AsyncLoader<object> {}
+const Refresh = () => {
+	const navigate = useNavigate();
+	useEffect(() => navigate(0), []);
+	return null;
+};
+
+const DeleteHolidayStop = (props: {
+	friendlyDateRange: string;
+	subscriptionName: string | undefined;
```

Comment:
I've maybe missed something but I am curious as to in what scenario would these be undefined? I would have thought that we would throw an error if they were undefined

Reply:
good question - in the old version it was an optional prop and it's type is basically coming from that

Reply:
Is it something that we want to keep or should we do a further refactor (not part of this PR) to have error handling?